### PR TITLE
upgrade to node 8

### PR DESCRIPTION
**Pull request by scisco**
This is needed to support testing cumulus packages in node 8 environment

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM circleci/node:6.10 
+FROM circleci/node:8.11
```
Comment (laurenfrederick) [inline Dockerfile:1]: Has this been tested somehow?

**Review by laurenfrederick — COMMENTED** [review]
Has

File: Dockerfile (hunk 1 above)
Comment (scisco) [reply Dockerfile:1]: yeah, all the circleci tests on the cumulus repo, use this image.

**Comment by scisco** [comment]
this is handled in #5

**Pull request closed without merging**